create_certs/Bumper_SAN.txt

```diff
@@ -3,4 +3,8 @@ ecovacs.com
 ecouser.net
 *.ecouser.net
 ecovacs.net
-*.ecovacs.net
\ No newline at end of file
+*.ecovacs.net
+*.ww.ecouser.net
+*.dc-eu.ww.ecouser.net
```

Comment:
@bmartin5692 Can you please verify, which domains the app used in the us? :)

Reply:
If it's of any help, here are all of the DNS queries and TLS server names for requests my older Deebot 901 makes:

```
bigdata-international.ecovacs.com
recommender.ecovacs.com
gl-us-api.ecovacs.com
api-app.ecouser.net
mq-ww.ecouser.net
```

Reply:
@puddly your urls are already covered by the wildcards domain above

Reply:
Indeed they are. I'm located in the USA and and your v2 changes work fine with the latest version of the app on Android (thank you!), so either the app "upgrades" to different domains when communicating with newer-generation bots, or these existing domains in your PR are enough.